README.adoc

```diff
@@ -107,21 +109,22 @@ include::{common-includes}/gitclone.adoc[]
 BFFResource.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/frontend/src/main/java/io/openliberty/guides/bff/BFFResource.java[]
+include::finish/bff/src/main/java/io/openliberty/guides/bff/BFFResource.java[]
 ----
 
 // File 1
 microprofile-config.properties
 [source, text, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/frontend/src/main/resources/META-INF/microprofile-config.properties[]
+include::finish/bff/src/main/resources/META-INF/microprofile-config.properties[]
 ----
 
 In this section, you will create a REST API for SSE in the `bff` service. When a
 client makes a request to this endpoint, the initial connection between the
 client and server is established and the client is subscribed to receive events
-pushed from the server. Later in this guide, you will have the front-end client use this
-endpoint to subscribe to the events pushed from the `bff` service.
+pushed from the server. Later in this guide, you will have the client in  the
```

Comment:
**change**
Later in this guide, you will have the client in  the `frontend` service use this endpoint to subscribe to the events pushed from the `bff` service.
**to**
Later in this guide, the client in the `frontend` service uses this endpoint to subscribe to the events that are pushed from the `bff` service.